### ISSUE TITLE
Part of the picture disappeared when burned out

### ISSUE THREAD

**Issue by RomulRS** [arkypita/LaserGRBL#794]
Hello friends. Help understand what the problem is. Missing part of the picture in the middle.
![IMG_5239](https://user-images.githubusercontent.com/33111677/69754213-69f19d80-1166-11ea-8122-46ba9f17c070.JPG)
![special-logo — копия](https://user-images.githubusercontent.com/33111677/69754214-6a8a3400-1166-11ea-8d37-63d23210962e.png)
![Безымянный](https://user-images.githubusercontent.com/33111677/69754215-6a8a3400-1166-11ea-905f-2a9f52a90095.png)

**Comment by arkypita** [comment]
This is really, really incredible. I have no idea how it could have happened.
I assume it is a hardware problem and only you can understand and tell us what it was. You should follow the process visually and figure out what jams when it passes through that area.

**Comment by brakthehun** [comment]
Can you upload the cut file??

**Comment by RomulRS** [comment]
> Can you upload the cut file??
Upload to? A laser engraver or a forum here?

> This is really, really incredible. I have no idea how it could have happened.
> I assume it is a hardware problem and only you can understand and tell us what it was. You should follow the process visually and figure out what jams when it passes through that area.
I observed that the laser in that place, which disappeared on one axis correctly, but on the other stood in place. It was as if voltage did not go to one motor for a while, or there was a program failure along the coordinates of one axis.
I noticed that the laser moved along one line for a long time back and forth. As if the voltage did not reach one engine for a while, or the program crashed along the coordinates of one axis.
